### PR TITLE
Fix images on production

### PR DESCRIPTION
**Pull request by dovganych**
**CHECK LIST**
- [ ] specs added
- [x] specs passed
- [x] code coverage >90%
- [ ] DB indexes added if needed
- [ ] seed.rb file is updated if needed
- [x] PR is reviewed manually again (to make sure you have 100% ready code)
- [x] PR doesn't have conflicts with main branch
- [ ] I've checked new feature as logged in and logged out user if needed
- [ ] "let!" is used in specs
- [x] screenshot of rails_best_practices and other tools added
- [x] screenshot of feature is added if task has UI
companies
company

empty_pdf

### CODE REVIEW

File: app/views/account/companies/show.html.slim [hunk 1]
```diff
@@ -66,51 +66,61 @@
...
-                  img.icon-right src="/assets/icon/google.png" Google
+                  = image_tag("icon/google.png", class: "icon-right")
+                  span Google
                 .card-text.col-sm-8
                   a href=@company.google target="_blank"=  @company.google
               .row
                 .card-text.col-sm-2
-                  img.icon-right src="/assets/icon/klout.png" Klout
+                  = image_tag("icon/klout.png", class: "icon-right")
+                  span Klout
```
Comment (Kmalynss) [inline app/views/account/companies/show.html.slim]: A lot of duplication here. Can it be DRYed in some way?

File: app/uploaders/company_logo_uploader.rb [hunk 2]
```diff
@@ -1,9 +1,13 @@
 class CompanyLogoUploader < CarrierWave::Uploader::Base
+  include CarrierWave::MiniMagick
+
+  process resize_and_pad: [150, 150, :transparent, "Center"]
```
Comment (igorkasyanchuk) [inline app/uploaders/company_logo_uploader.rb]: WRONG
create a different version

NEVER resize original images

File: app/uploaders/company_logo_uploader.rb [hunk 3]
```diff
@@ -1,9 +1,13 @@
...
+
   def store_dir
     "uploads/#{model.class.to_s.underscore}/#{mounted_as}/#{model.id}"
   end
 
   def default_url
-    "/assets/" + [version_name, "default.png"].compact.join('_')
+    "fallback/default.png"
```
Comment (igorkasyanchuk) [inline app/uploaders/company_logo_uploader.rb]: revert
and create an appropriate fallback  version

File: app/views/account/companies/show.pdf.slim [hunk 4]
```diff
@@ -7,8 +7,9 @@
         .card-content.collapse.show
           .card-body.container
             .row
-              .col-3
-                img.img-fluid [src="#{company.logo.path}" alt="Company Logo"]
+              - if company.logo.path
```
Comment (igorkasyanchuk) [inline app/views/account/companies/show.pdf.slim]: path?

try to google how to do it correctly

File: app/views/account/companies/show.pdf.slim [hunk 5]
```diff
@@ -7,8 +7,9 @@
...
+                .col-3
+                  = image_tag(company.logo.path, alt: company.name, class: "img-fluid")
```
Comment (igorkasyanchuk) [inline app/views/account/companies/show.pdf.slim]: again "path". incorrect. use tutorial

**Pull request merged**